addons/mail/static/src/components/chatter/tests/chatter_tests.js

```diff
@@ -138,6 +138,11 @@ QUnit.test('base rendering when chatter has no record', async function (assert)
         '.o_MessageList_loadMore',
         "should not have the 'load more' button"
     );
+    assert.containsNone(
+        document.body,
+        '.o_MessageActionList',
+        "should not have action list in message"
+    );
```

Comment:
I don't think this test covers the bug?

There is no `o_MessageActionList` by default because it only shows on hover (or on click), so that assertion would work even without the fix.